packages/common/src/services/utilities.ts

```diff
@@ -193,7 +193,10 @@ export function unflattenParentChildArrayToTree<P, T extends P & { [childrenProp
   // make them accessible by guid on this map
   const all: any = {};
 
-  inputArray.forEach((item: any) => all[item[identifierPropName]] = item);
+  inputArray.forEach((item: any) => {
+    all[item[identifierPropName]] = item;
+    delete item[childrenPropName];
```

Comment:
the only thing I wonder is why are you deleting **after** and not **before** the assignment, so why not inverse lines 197 & 198 ?

Reply:
These lines are interchangeable without any affect on each other.
 
Line 197 maps the item reference under the set id field.
Line 198 removes the children property from the item. 

I've added the line here as it was already a place where all items were iterated already.

Reply:
ok thanks, let's merge and hope this fixes it all :)